images/build/yarn-cache/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # It is never used directly, but rather as a base for our build image.
 # Yarn cache will be placed in /usr/local/share/yarn-cache, that is the folder
 # that must be copied to the build image.
-FROM node:lts
+FROM node:10.16.3
```

Comment:
This was changed because `lts` now points to `12.13.0` which is not part of this PR. I am sending out one more PR for updating all versions.